pkg/translator/prometheusremotewrite/helper_test.go

```diff
@@ -250,14 +253,57 @@ func Test_createLabelSet(t *testing.T) {
 				res.Attributes().PutBool("service.instance.id", true)
 				return res
 			}(),
+			pcommon.NewInstrumentationScope(),
 			lbs1,
 			map[string]string{},
 			[]string{label31, value31, label32, value32},
 			getPromLabels(label11, value11, label12, value12, label31, value31, label32, value32, "job", "12345", "instance", "true"),
 		},
+		{
+			"labels_with_scope",
```

Comment:
There doesn't seem to be a test for a scope with name but without version, or the inverse.  My read of [the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-scope) is that name is required and version is not (though I'd give a strong side eye to anyone not including one).